server/data/interfaces/prisonApiClient.ts

```diff
@@ -3,11 +3,15 @@ import { Location } from './location'
 import { BlockRollCount } from './blockRollCount'
 import { Movements } from './movements'
 import { StaffRole } from './staffRole'
+import { OffenderCell } from './offenderCell'
 
 export interface PrisonApiClient {
   getUserCaseLoads(): Promise<CaseLoad[]>
   getUserLocations(): Promise<Location[]>
   getRollCount(prisonId: string, unassigned?: boolean): Promise<BlockRollCount[]>
```

Comment:
I know this is existing code but maybe worth presenting an object to this method instead of a boolean flag?

So `getRollCount({ prisonId: `LEI', unassigned: true })`

rather than `getRollCount('LEI', true)` <- bit harder to read because you then have to look at the function definition to work out what true means

Reply:
Have done this.